vm/message/src/lib.rs

```diff
@@ -63,8 +63,8 @@ impl Message for ShimMessage {
     fn params(&self) -> &RawBytes {
         &self.params
     }
-    fn gas_limit(&self) -> i64 {
-        self.gas_limit as i64
+    fn gas_limit(&self) -> u64 {
+        self.gas_limit
     }
     fn set_gas_limit(&mut self, token_amount: i64) {
```

Comment:
This should be changed as well.
```suggestion
    fn set_gas_limit(&mut self, token_amount: u64) {
```

Reply:
Fixed